# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/build/FileLocation.cpp
# Commit: 24ee1c0756ae
# Full Hash: 24ee1c0756ae49b467d53c499dd0c16839e05eb2
# Author: Randell Jesup <rjesup@wgate.com>
# Date: 2022-03-08 22:01:59
# Regressor Bug: 1744043
# File Overlap Count: 1
# Description:
#   Bug 1744043: Clean up nsJAR r=nika,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132794
# ==============================================================================

diff -r 1c746ea21b95 -r 24ee1c0756ae xpcom/build/FileLocation.cpp
--- a/xpcom/build/FileLocation.cpp	Tue Mar 08 13:05:28 2022 +0000
+++ b/xpcom/build/FileLocation.cpp	Tue Mar 08 13:57:56 2022 +0000
@@ -151,8 +151,7 @@
   }
   aData.mZip = mBaseZip;
   if (!aData.mZip) {
-    aData.mZip = new nsZipArchive();
-    aData.mZip->OpenArchive(mBaseFile);
+    aData.mZip = nsZipArchive::OpenArchive(mBaseFile);
   }
   aData.mItem = aData.mZip->GetItem(mPath.get());
   if (aData.mItem) {